src/content/developers/docs/apis/json-rpc/index.md

```diff
@@ -386,7 +431,7 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"eth_chainId","params":[],"id":67
 
 ### eth_mining {#eth_mining}
 
-Returns `true` if client is actively mining new blocks.
+Returns `true` if client is actively mining new blocks. This can only return `true` for proof-of-work networks and may not be available in some clients since The Merge to proof of stake..
```

Comment:
I don't think it's correct to say 'The Merge to proof-of-stake'. 

It should either be: 

1. "...since The Merge."
2. "...since the transition to proof-of-stake."